### DISCUSSION TITLE
Literal.ts, paramento valor.

### DISCUSSION THREAD

**Discussion by ItaloCobains** [DesignLiquido/delegua#155] [P&R] [answered]
### Literal.ts

Estou com algumas dificuldade ao tipar o código pois o parâmetro valor na classe Literal esta como any e em outras classes tbm.
Não tenho ideia de qual pode ser o tipo dele.

@leonelsanchesdasilva tem alguma ideia ?

**Comment by leonelsanchesdasilva** [comment, marked as answer]
@ItaloCobains O valor neste caso é composto. Eu faria algo como:

`valor: number | string | number[] | string[] | any`

O `any` no final é por causa do valor `nulo`.
